src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
```

Comment:
Don't use unnecessary breaks between code and don't create temporary variables if you can directly return value from the method and don't use "magic" numbers. In that way your code will be easier to read and modify. 
```suggestion
    private final int BOUND = 100;

    public Ball getRandomBall() {
        return new Ball(
                getRandomColor(),
                new Random().nextInt(BOUND));
    }
```